### ISSUE TITLE
qcow2 copy on write

### ISSUE THREAD

**Issue by Eeems** [timower/rM-docker#14]
It would be interesting to use copy on write to store changes to the base image in a way where this can be uploaded without the rootfs, but with the qcow2 images with all the changes from the base image. Then fetch and create the base qcow2 image if it's missing on launch of the container.

**Comment by timower** [comment]
Not sure I understand. You want to distribute the deltas, so other people can re-use it somehow, without distributing the proprietary base image?

**Comment by Eeems** [comment]
> Not sure I understand. You want to distribute the deltas, so other people can re-use it somehow, without distributing the proprietary base image?

Exactly. That way, people can cache portions of setup that take a long time, like installing all the dev tools required to compile stuff on the device or something similar.
